sdcm/tester.py

```diff
@@ -272,11 +273,25 @@ def __init__(self, *args):  # pylint: disable=too-many-statements
         time.sleep(0.5)
         InfoEvent('TEST_START test_id=%s' % Setup.test_id())
 
+    def _init_timeout_thread(self) -> threading.Timer:
```

Comment:
I think it's not a thread, it's a timer.. (even if the implementation might be a thread)

Reply:
@fruch , you are right, but there is one goood point it keeping it as it is - it will be easier to locate service threads when we finaly will decide to implement single framework for them.

Reply:
`_init_timeout_thread` -> `_init_test_timeout_thread`